src/theia/sfm/pose/four_point_focal_length_radial_distortion.cc

```diff
@@ -0,0 +1,632 @@
+// This file was created by Steffen Urban (urbste@googlemail.com) or company address (steffen.urban@zeiss.com)
+// December 2018
+
+
+#include <Eigen/Core>
+#include <Eigen/SVD>
+#include <Eigen/Dense>
+#include <Eigen/Geometry>
+#include <Eigen/LU>
+#include <Eigen/QR>
+#include <Eigen/Cholesky>
+#include <Eigen/Eigenvalues>
+#include <random>
+
+#include "theia/util/random.h"
+#include "theia/sfm/pose/four_point_focal_length_radial_distortion.h"
+
+namespace theia
+{
+
+theia::RandomNumberGenerator rng(42);
```

Comment:
no global variables please -- you can make this a static var inside the function where it is needed